receiver/kubeletstatsreceiver/metadata.yaml

```diff
@@ -185,6 +185,13 @@ metrics:
     gauge:
       value_type: double
     attributes: [ ]
+  k8s.pod.cpu.usagePercent:
+    enabled: false
```

Comment:
Can we enable these by default?

Reply:
New metrics should be added as optional initially. See https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/scraping-receivers.md#changing-the-emitted-metrics

Reply:
I'm not sure about the naming. I don't believe we have `camelCase` recommended anywhere in sem conv. The spec also recommends using [0,1] ratio instead. I was thinking of making it configurable generally in the metrics builder tho. e.g via `scale` option

Reply:
Do you mean update mdatagen to allow the cpu.utilization and memory.usage to switch back and forth between the raw value and percentage value? Id prefer multiple metrics as it is useful to know both the exact value and how that compares to the limit.

Reply:
I am open to any name changes

Reply:
> Do you mean update mdatagen to allow the cpu.utilization and memory.usage to switch back and forth between the raw value and percentage value?

No, only ratio to percentage conversion for the same metric. As I mentioned, the ratio [0, 1] is what OTel spec recommends instead of percentage, so we need to add these metrics in [0,1] scale by default

Reply:
Oh ok I'm following. I can make those changes for these 4 new metrics if you want

Reply:
Well I think I'm following. Do you want me to stop multiplying these new values by 100 and force them between 0 and 100?

Reply:
> Do you want me to stop multiplying these new values by 100 and force them between 0 and 100?

These metrics have to be in the [0,1] range, not [0,100] according the the OTel spec

Reply:
Ya sorry I typoed.

Do you have a link to the section in the spec that specifies gauges should use [0,1] ratios instead of percentages? I haven't been able to find it.

Reply:
Is it here: https://github.com/open-telemetry/opentelemetry-specification/blob/f20566d6651941a10188c3eb2c0ea4519d5d7be0/specification/metrics/semantic_conventions/README.md#instrument-naming?

Reply:
> Is it here: https://github.com/open-telemetry/opentelemetry-specification/blob/f20566d6651941a10188c3eb2c0ea4519d5d7be0/specification/metrics/semantic_conventions/README.md#instrument-naming?

Yes

Reply:
In that case the receiver's `*.cpu.utilization` metric is probably breaking the experimental semantic convention. Based on that section's description for `usage` vs `utilization`, the existing `*.cpu.utilization` should be ``*.cpu.usage`, like the memory metrics and these new metrics should be `*.[cpu|memory].utilization`.

Reply:
Sounds reasonable to me. `memory.usage` is already there, so we only need to change CPU metrics. I'm worried that it'll be a significant breaking change as the metrics must be used extensively in the field. We can put it under a feature gate or configuration option for an extended period. I'd wait until we stabilize container/k8s metrics in the spec before enabling it by default.

Reply:
Ya I agree using feature gates and waiting for the spec finalization is important here. @mx-psi has the topic of utilization vs usage come up in the system metrics conversations?

Reply:
@dmitryax since this is not a breaking change for memory, how about I close this PR and open one only to add memory.utilization? Then I'll open another for CPU with feature gates and we can have the hard discussion about the impacts and breaking changes there.

Reply:
Given that the `utilization` metrics will require additional k8s API calls, we should keep them optional going forward along with `cpu.utilization`. So we can probably start by disabling them before applying the feature gate. Changing optional metrics is less disruptive.

If it makes sense to you, the first PR would be to add an `if_enabled_not_set` warning to the cpu utilization metrics with a message like "This metric will be disabled soon. After that, it'll be changed to report the ratio of usage/limit applied via a feature gate. Use container.cpu.usage metric instead."

Reply:
Also please take a look at the metrics dockerstats metrics. We need to be consistent with them